src/main/webapp/js/user-page-loader.js

```diff
@@ -39,6 +39,9 @@ function showMessageFormIfLoggedIn() {
       })
       .then((loginStatus) => {
         if (loginStatus.isLoggedIn) {
+            if (loginStatus.username == parameterUsername){
```

Comment:
Alignment looks a bit off. Shouldn't the `if` on `line 42` align with the `const` on `line 45`?